ui_framework/doc_site/src/views/tabs/tabs_example.js

```diff
@@ -11,6 +12,14 @@ import {
 const html = require('./tabs.html');
 const js = require('raw!./tabs.js');
```

Comment:
Would you mind deleting the original example files and removing the example from this file?